### PR TITLE
'solution'

### PR DESCRIPTION
**Pull request by IvanGuculyak**

### CODE REVIEW

**Comment by IvanGuculyak** [comment]
print_screen

File: static/css/styles.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+body {
+    margin-top: 20px;
+}
+
+.sidebar-nav {
+    padding: 0;
+    list-style: none;
+}
```
Comment (ArsenPidhoretskyi) [inline static/css/styles.css:8]: Add empty line

File: templates/base.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+          {% include "includes/sidebar.html" %}
+        {% endblock %}
+      </div>
+      <div class="col-sm-9">
+        {% block content %}{% endblock %}
+      </div>
+    </div>
+  </div>
+</body>
+</html>
```
Comment (ArsenPidhoretskyi) [inline templates/base.html:29]: Add empty line

File: templates/includes/sidebar.html [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<ul class="sidebar-nav">
+  <li><a href="{% url 'taxi:index' %}">Home Page</a></li>
+  <li>Manufacturers</li>
+  <li>Cars</li>
+  <li>Drivers</li>
+</ul>
```
Comment (ArsenPidhoretskyi) [inline templates/includes/sidebar.html:6]: Add empty line

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

File: static/css/styles.css (hunk 1 above)
Comment (vasylhnatiuk) [reply static/css/styles.css]: do it

**Review by vasylhnatiuk — CHANGES_REQUESTED** [review]

**Review by vasylhnatiuk — APPROVED** [review]

**Pull request closed without merging**